docker/Dockerfile.dev

```diff
@@ -3,9 +3,8 @@ LABEL MAINTAINER="BJ Dierkes <derks@datafolklabs.com>"
 ENV PS1="\[\e[0;33m\]|> cement-py312 <| \[\e[1;35m\]\W\[\e[0m\] \[\e[0m\]# "
 
 WORKDIR /src
-COPY requirements-dev.txt /src/
-RUN apk update \
-    && apk add libmemcached-dev \
+RUN apk update --no-check-certificate \
```

Comment:
The docker files are still in draft state - I just added this to make them work in my environment, will remove the no-check-certificate again